schema/resolvers.js

```diff
@@ -175,7 +175,13 @@ const Resolvers = {
   Questionnaire: {
     sections: (questionnaire, args, ctx) =>
       ctx.repositories.Section.findAll({ QuestionnaireId: questionnaire.id }),
-    createdBy: questionnaire => ({ name: questionnaire.createdBy })
+    createdBy: questionnaire => ({ name: questionnaire.createdBy }),
+    questionnaireInfo: ({ id }) => id
```

Comment:
Is `questionnaireInfo` field required here? If it's omitted would the `QuestionnaireInfo` resolver still work like this?

```js
QuestionnaireInfo: {
    totalSectionCount: ({id: questionnaireId}, args, ctx) =>
      ctx.repositories.Section.getSectionCount(questionnaireId)
}
```

Reply:
If omitted it doesn't resolve to `QuestionnaireInfo` and response is:

```
{
  "data": {
    "section": {
      "id": "4",
      "title": "<p>S1</p>",
      "description": "",
      "position": 0,
      "__typename": "Section",
      "questionnaire": {
        "questionnaireInfo": null
      }
    }
  }
}
```